README.md

```diff
@@ -95,6 +95,29 @@ module.exports = {
 }
 ```
 
+### Verbose output:
+You can enable verbose output to check which files have been processed by the replace loader.
+In your `webpack.config.js`:
+
+```javascript
+module.exports = {
+  // ...
+  module: {
+    rules: [
+      {
+        test: /filename\.js$/,
+        loader: 'improved-pattern-replace-loader',
```

Comment:
```suggestion
        loader: 'pattern-replace-loader',
```

Reply:
Yes, I have enabled that maintainers can edit the PR. The readme is different because I created a fork of it. Just do modify the loader name when merging it. For my fork the above loader comment is correct can you please fix it during merge?

Reply:
I don't believe so. This will change my original package, therefore we should keep it the same.

Reply:
After we merge, I'm going to create a new version with your changes and then you should be able to use that version on your project.

Reply:
This sounds like a plan :). Feel free to edit the loader name.

Reply:
Please remove the `package.json` changes expect for the version bump up and make it `1.0.5` so we are able to merge it.